llvm/utils/gn/secondary/llvm/lib/Support/BUILD.gn

```diff
@@ -33,6 +33,7 @@ static_library("Support") {
     "Windows",
   ]
   sources = [
+    "AArch64BuildAttributes.cpp"
```

Comment:
(fyi, there's no need to update .gn files. They're updated automatically by a bot based on changes to cmake files. if you do update them, please add trailing commas and add files just once instead of twice like here. i fixed this in 92b839e9c82450a3c465d349de73818e6aad59f3)